NEWS

```diff
@@ -1,3 +1,7 @@
+### mate-desktop 1.25.0
+
+  * a11y-keyboard: Add capslock-beep-enable setting
```

Comment:
Can you only add new version here?

Adding Commit history will be done with the release commit when releasing.

Reply:
OK, but without this Travis fails because distcheck fails.

Reply:
No i mean add only the version to NEWS.
```
### mate-desktop 1.25.0


### mate-desktop 1.24.0
```
Distcheck requires only a version number.
When this is merged, i will make a new release to make your other PRs happy.

Reply:
For better understanding.
For releasing a tarball i need a minimum change (adding git log entries to NEWS) to generate a commit which can be tagged.
A commit + tag will start the machinery at travis, github and our website to create a new release with tarball.

Reply:
OK got it, I removed the list item.